pimd/pim_cmd.c

```diff
@@ -11180,3 +11298,10 @@ void pim_cmd_init(void)
 	install_element(INTERFACE_NODE, &no_ip_pim_bfd_param_cmd);
 #endif /* !HAVE_BFDD */
 }
+
+void pim6_cmd_init(void)
```

Comment:
This doesn't belong in here, this PR should just make the "IPv4 preparation" changes without adding any IPv6 bits yet.